sibr-cassandra-migrations.cql

```diff
@@ -1,5 +1,11 @@
 CREATE KEYSPACE IF NOT EXISTS blaseball WITH replication = { 'class': 'SimpleStrategy', 'replication_factor': 1 };
 
+CREATE TABLE IF NOT EXISTS blaseball.imported_logs(
+  id uuid PRIMARY KEY,
+  key text,
```

Comment:
What is `key` here? Or has that not been determined yet

Reply:
It's the S3 object key in iliana's bucket.